src/scripts/main.js

```diff
@@ -1,3 +1,77 @@
 'use strict';
 
 // write code here
+
+const buttons = document.querySelector('.container');
```

Comment:
it is better to you more descriptive name for this var since your container is not basically buttons

Reply:
not fixed